easybuild/easyblocks/s/samtools.py

```diff
@@ -76,7 +81,10 @@ def __init__(self, *args, **kwargs):
             self.include_files += ["sam_header.h"]
             self.bin_files += ["misc/varfilter.py"]
 
-        self.lib_files = ["libbam.a"]
+        if LooseVersion(self.version) < LooseVersion('1.14'):
+            # bam_endian.h and sam.h removed from 1.14
+            self.include_files += ["bam_endian.h", "sam.h"]
+            self.lib_files = ["libbam.a"]
```

Comment:
```suggestion

        self.lib_files = ["libbam.a"]
```
otherwise old versions no longer install (or sanity check for) `libbam.a`